### PR TITLE
feat(pie-modal): add basic styling for the opened state

### PR DESCRIPTION
**Pull request by fernandofranca**
## @justeattakeaway/pie-modal

- [Added] - Basic styling for modal default state
- [Added] - Modal heading added to component
- [Updated] - Added modal component to storybook file

## @justeattakeaway/pie-wrapper-react

[Changed] - Custom Element Definitions

## pie-monorepo

[Updated] - yarn.lock

### CODE REVIEW

File: packages/components/pie-modal/src/index.ts [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 export class PieModal extends RtlMixin(LitElement) {
     // eslint-disable-next-line class-methods-use-this
     render () {
         return html`
-            <div>This is the Pie Modal
-                <slot></slot>
-            </div>`;
+            <dialog class="o-modal" open>
+                <h3 class="o-modal-header">Modal header</h3>
+                <div class="o-modal-content-wrapper">
```
Comment (ashleynolan) [inline packages/components/pie-modal/src/index.ts:12]: Nit – for the naming, would be `contentWrapper` (as dashes show an extra level of DOM hierarchy)

```suggestion
                <div class="c-modal-contentWrapper">
```

File: .changeset/polite-waves-buy.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"pie-storybook": minor
+---
+
+[Updated] - PieModal story file
```
Comment (ashleynolan) [inline .changeset/polite-waves-buy.md:5]:
```suggestion
[Changed] - Added modal component to storybook file
```

File: .changeset/new-actors-wave.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-wrapper-react": minor
+---
+
+[Updated] - Definitions
```
Comment (ashleynolan) [inline .changeset/new-actors-wave.md:5]:
```suggestion
[Changed] - Custom Element Definitions
```

File: .changeset/strong-doors-walk.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"pie-monorepo": patch
+---
+
+[Updated] - yarn.lock
```
Comment (ashleynolan) [inline .changeset/strong-doors-walk.md:5]:
```suggestion
[Changed] - yarn.lock
```

File: packages/components/pie-modal/src/index.ts [hunk 5]
```diff
@@ -1,14 +1,23 @@
...
+
 export class PieModal extends RtlMixin(LitElement) {
     // eslint-disable-next-line class-methods-use-this
     render () {
         return html`
-            <div>This is the Pie Modal
-                <slot></slot>
-            </div>`;
+            <dialog class="o-modal" open>
+                <h3 class="o-modal-header">Modal header</h3>
```
Comment (ashleynolan) [inline packages/components/pie-modal/src/index.ts:11]: I'd probably go with `heading` - `header` could be used if we ever have a header block for the modal in future. 

```suggestion
                <h3 class="o-modal-heading">Modal header</h3>
```

File: .changeset/good-drinks-compare.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+"@justeattakeaway/pie-modal": minor
+---
+
+[Added] - Header
+[Added] - Basic styling
```
Comment (ashleynolan) [inline .changeset/good-drinks-compare.md:6]:
```suggestion
[Added] - Modal heading added to component
[Added] - Basic styling for modal default state
```

**Review by LTurns — APPROVED** [review]

**Review by ashleynolan — COMMENTED** [review]
Looks good now – just the one comment around CSS Variables otherwise :shipit:

**Review by LTurns — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

File: packages/components/pie-modal/src/modal.scss [hunk 7]
```diff
@@ -0,0 +1,53 @@
+@use '@justeat/pie-design-tokens/dist/jet.scss' as dt;
+
+$modal-size-l: 1080px;
```
[the remark this answers is not in the document]
Comment (ashleynolan) [reply packages/components/pie-modal/src/modal.scss:1]: I blame @LTurns and @kevinrodrigues for approving 😆

**Pull request merged**